pom.xml

```diff
@@ -680,12 +478,6 @@
           </configuration>
           <executions>
             <execution>
-              <id>integration-test</id>
```

Comment:
Why was the `integration-test` goal removed?
https://maven.apache.org/surefire/maven-failsafe-plugin/usage.html